cluster-autoscaler/cloudprovider/equinixmetal/manager_rest.go

```diff
@@ -695,7 +706,9 @@ func (mgr *equinixMetalManagerRest) templateNodeInfo(nodegroup string) (*schedul
 	node.ObjectMeta = metav1.ObjectMeta{
 		Name:     nodeName,
 		SelfLink: fmt.Sprintf("/api/v1/nodes/%s", nodeName),
-		Labels:   map[string]string{},
+		Labels:   map[string]string{
+			// apiv1.LabelHostname: nodeName,
```

Comment:
I want to think we can add `LabelHostname` here, but I'm not sure we need to (and I'm not sure if nodeName matches hostName).